src/app/stream-deck/async-module.ts

```diff
@@ -26,24 +27,18 @@ export async function sendToStreamDeck(msg: SendToStreamDeckArgs) {
   }
 }
 
-const installFarmingObserver = _.once(() => {
-  const state = store.getState();
-  // send initial state
-  sendToStreamDeck({
-    action: 'farmingMode',
-    data: Boolean(state.farming.storeId),
-  });
-  // observe farming mode state
-  observeStore(
-    (state) => state.farming.storeId,
-    (_, newState) => {
+function createFarmingObserver(): StoreObserver<string | undefined> {
```

Comment:
Not sure why this would need to be a function

Reply:
Just a convention I have had in other instances.